micro-rdk/src/common/app_client.rs

```diff
@@ -257,6 +257,7 @@ impl AppClient {
             version: env!("CARGO_PKG_VERSION").to_string(),
             git_revision: "".to_string(),
             platform: Some("esp32".to_string()),
+            platform_tags: Vec::new(),
```

Comment:
Is `vec!([])` preferred?

Reply:
they both resolve to pretty much the same thing by the compiler for an empty vec, at least for most platforms. 
updated it to `vec![]`.